### DISCUSSION TITLE
company management

### DISCUSSION THREAD

**Discussion by nagy1991** [filamentphp/filament#6691] [Help]
Dear support!
I would like to ask for help with the following.
There are companies and users belong to them. A user can be in several companies. But there are users who do not belong to your company, but because of their role, they can see all the lists, etc. How to solve this?

**Comment by andrewdwallo** [comment]
https://github.com/andrewdwallo/erpsaas/tree/1.x maybe this can help. I am still developing. But I use my package filament companies in it.
